prime-router/src/main/kotlin/fhirengine/translation/HL7toFhirTranslator.kt

```diff
@@ -77,6 +69,9 @@ class HL7toFhirTranslator internal constructor(
         // https://github.com/LinuxForHealth/hl7v2-fhir-converter/blob/d5e43fffa96654e7c5bc896e020ff2fa8aac4ff2/src/main/java/io/github/linuxforhealth/hl7/HL7ToFHIRConverter.java#L135-L159
         // If timezone specification is needed it can be provided via a custom HL7MessageEngine with a custom FHIRContext that has the time zone ID set
 
+        // reinitialize the ResourceReader to ensure the correct singleton instance for the job is loaded
+        ResourceReader(ConverterConfiguration(configFolderPath))
```

Comment:
In a number of places processing switches from an local instance of ResourceReader to the singleton; reinitializing here to make sure the two are equivalent.

Reply:
This seems a bit off?  Is there a unit test that would fail if I uncommented this line out?